robottelo/cli/contentview.py

```diff
@@ -78,6 +78,18 @@ def version_info(cls, options):
         return result
 
     @classmethod
+    def version_inc_update(cls, options):
+        """Performs incremental update of the content-view's version"""
+        cls.command_sub = 'version incremental-update'
+
+        if options is None:
+            options = {}
+
+        result = cls.execute(cls._construct_command(options))
+        result.stdout = hammer.parse_info(result.stdout)
```

Comment:
Would be better to parse a new output format on https://github.com/SatelliteQE/robottelo/blob/master/robottelo/common/ssh.py#L25-L28. If output format is 'info' then parse the info.

This method will become something like:

```python
cls.command_sub = 'version incremental-update'
if options is None:
    options = {}
return cls.execute(cls._construct_command(options), output_format='info')
```